device_operation/include/device_gemm_xdl.hpp

```diff
@@ -12,6 +12,8 @@
 #include "tensor_descriptor_helper.hpp"
 #include "gridwise_gemm_xdlops_v2r3.hpp"
 
+#define USE_M_N_Padding 1
```

Comment:
Please use a template argument to control this.


See ConvForwardSpecialization for reference
https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/823657ed120144943b7db87c07fe3e647128db56/device_operation/include/device_conv2d_fwd_xdl_c_shuffle_nhwc_kyxc_nhwk.hpp#L122-L123